src/scope/network/http/http.ts

```diff
@@ -129,16 +129,16 @@ export class Http implements Network {
   static async getNetworkConfig(): Promise<NetworkConfig> {
     const obj = await list();
 
-    const strictSSL = obj[CFG_NETWORK_STRICT_SSL] ?? obj[CFG_PROXY_STRICT_SSL]
+    const strictSSL = obj[CFG_NETWORK_STRICT_SSL] ?? obj['network.strict_ssl'] ?? obj[CFG_PROXY_STRICT_SSL]
```

Comment:
why you are not using here `obj[CFG_NETWORK_STRICT_SSL]` after the `??`

Reply:
I use it as the beginning. Then I check the old setting name.